src/bentoml/start.py

```diff
@@ -100,12 +98,16 @@ def start_runner_server(
                     )
                     break
                 else:
-                    triton_handle = construct_triton_handle(runner, **attrs)
+                    cli_args = runner.cli_args + [
+                        f"--http-port={runner.protocol_address.split(':')[-1]}"
+                        if runner.tritonserver_type == "http"
+                        else f"--grpc-port={runner.protocol_address.split(':')[-1]}"
```

Comment:
If users happen to specify `--http-port` or `--grpc-port`, will they override the ones defined here?

Reply:
no, because it appears at the end, so our options should always override users.